static/css/restyle/restyle.less

```diff
@@ -950,10 +950,6 @@ a.button.release-theme-lock {
   }
 }
 
-.button:nth-child(n+1) {
```

Comment:
This was added when we had all the many buttons which turned out to be a bug so this removes it since it adds a margin unecessarily and does so even when adjacent to a hidden button.